bpf/tc/attach.go

```diff
@@ -486,14 +555,20 @@ func HasQdisc(ifaceName string) (bool, error) {
 }
 
 // RemoveQdisc makes sure that there is no qdisc attached to the given interface
-func RemoveQdisc(ifaceName string) error {
+func RemoveQdisc(ifaceName, hook string) error {
```

Comment:
The qdisc is shared by both directions so removing the qdisc implicitly removes _both_ directions of program.  I don't think it's right to have a hook param here.